Source/WebGPU/WGSL/Overload.cpp

```diff
@@ -143,6 +143,9 @@ Type* OverloadResolver::materialize(const AbstractType& abstractType) const
         },
         [&](TypeVariable variable) -> Type* {
             auto* resolvedType = resolve(variable);
+            if (!resolvedType)
+                return nullptr;
+
```

Comment:
This looks misplaced. Are you sure this shouldn't be in a separate patch?

Reply:
Thanks, that should not be in this patch